docker-compose.yml

```diff
@@ -74,6 +78,8 @@ services:
       - 52000:3000
 
 volumes:
+  uwazi_uploaded_documents:
```

Comment:
Using this name makes the final name be like

```
$ docker volume ls
local               uwazi-docker_esdata1
local               uwazi-docker_uwazi_uploaded_document
```

Reply:
What is important is that both are easily accessible from the host for easy backup / restore / migration. Woudn't a bind mount be better in that case than a named volume? Sure, on Linux hosts, you can access the named volume if you have root access, but you don't have to search for it, if you use a bind mount. Anyway, the main thing is that persistent data does not get stored inside the container, which may be transient.

Reply:
Also, if you already have some data on any container (like some old one before this PR is ready), this is one way to export https://docs.docker.com/engine/reference/commandline/export/.

When I wake up and test more the PR I will also document a bit more

Reply:
I think it will work. But in addition to rename it, I will test a bit more later, document and then merge to master.

```sh
# fititnt at bravo in /alligo/code/fititnt/uwazi-docker on git:bug-uploaded_documents o [6:17:00]
$ docker volume inspect uwazi-docker_uwazi_uploaded_documents           
[
    {
        "CreatedAt": "2018-10-16T06:14:15-03:00",
        "Driver": "local",
        "Labels": {
            "com.docker.compose.project": "uwazi-docker",
            "com.docker.compose.version": "1.21.2",
            "com.docker.compose.volume": "uwazi_uploaded_documents"
        },
        "Mountpoint": "/var/lib/docker/volumes/uwazi-docker_uwazi_uploaded_documents/_data",
        "Name": "uwazi-docker_uwazi_uploaded_documents",
        "Options": null,
        "Scope": "local"
    }
]

# fititnt at bravo in /alligo/code/fititnt/uwazi-docker on git:bug-uploaded_documents o [6:17:10]
$ sudo ls /var/lib/docker/volumes/uwazi-docker_uwazi_uploaded_documents/_data
1539679896696v678rq6k7gp.jpg  1539681252641g7pge4y1ijt.pdf  5bc5abe47e44ab001c2b87be.jpg

```

Reply:
OK, can't wait to test it (but maybe I should anyway, and do a bit of my _actual_ work in the meantime...

Reply:
Well, technically speaking, is possible to test a pull request without it having to be added to the master branch and copy and pasting changed likes manually. But another day can explain it. This one is going to be added soon.